content/en/developers/authorization/oauth2_in_datadog.md

```diff
@@ -28,23 +28,29 @@ This page provides a step-by-step overview on how to implement the OAuth protoco
+   - In order to build the URL for this POST request, use the `domain` query parameter that is provided on the redirect to your `onboarding_url`. 
    - This parameter is only provided if the user initiates authorization from the Datadog integration tile. See the [Initiate authorization from a third-party location](#Initiate-authorization-from-a-third-party-location) section for more options if the user chooses to initiate authorization externally.  
-   - For example, `site` may be `https://app.datadoghq.com`, `https://app.datadoghq.eu`, `https://us5.datadoghq.com`, or it might have a custom subdomain that must be accounted for, such as `https://<custom_subdomain>.datadoghq.com`. 
-   - To handle multiple Datadog sites dynamically, you can append the Datadog `authorize` path to this constructed URL.
+   - The `domain` query parameter provides the [Datadog site][17] that the authorizing user is in, and will be needed to construct the URL for this POST request to the Authorize endpoint: 
+       - For users authorizing from the US1, Fed1, and EU1 Datadog sites, the URL will be: `https://app.<domain>/oauth2/v1/authorize?...`
+       - For users authorizing from any other Datadog site, the URL will be: `https://<domain>/oauth2/v1/authorize?...`
+
+<div class="alert alert-info">The `domain` parameter is new as of July 2023. OAuth clients created before then use the `site` parameter instead. The `site` parameter will continue to be supported, but we encourage use of the `domain` parameter to make API calls more seamless. </div>
 
 4. Once a user clicks **Authorize**, Datadog makes a POST request to the authorize endpoint. The user is redirected to the `redirect_uri` that you provided when setting up the OAuth Client with the authorization `code` parameter in the query component.
 
 5. From the `redirect_uri`, make a POST request to the [Datadog token endpoint][10] that includes the authorization code from Step 4, the `code_verifier` from Step 3, your OAuth client ID, and client secret.
 
-   - In order to build the URL for this post request, use the `site` query parameter that is provided on the redirect to your `redirect_uri`. 
-   - For example, `site` might be `https://app.datadoghq.com`, `https://app.datadoghq.eu`, `https://us5.datadoghq.com`, or it might have a custom subdomain that must be accounted for, such as `https://<custom_subdomain>.datadoghq.com`. 
-   - To handle multiple Datadog sites dynamically, you can append the Datadog `token` path to this constructed URL.
+   - In order to build the URL for this post request, use the `domain` query parameter that is provided on the redirect to your `redirect_uri`. 
+   - The `domain` query parameter provides the [Datadog site][17] that the authorizing user is in, and will be needed to construct the URL for this POST request to the token endpoint: 
+       - For users authorizing from the US1, Fed1, and EU1 Datadog sites, the URL will be: `https://api.<domain>/oauth2/v1/token?...`
```

Comment:
@bgoldberg122 Just want to confirm that these are correct for the token endpoint

Reply:
These are the [docs](https://docs.datadoghq.com/developers/authorization/oauth2_endpoints/?tab=tokenendpoints#exchange-authorization-code-for-access-token). If you want more formal confirmation I would ask AAA

Reply:
Waiting on confirmation for this

Reply:
Finalized this with API platform team and updated docs accordingly